src/__tests__/Grid.Col.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { shallow } from 'enzyme';
+
+import { Col } from '../Grid';
+
+const MockComponent = props => <div css={props.styling} />;
+MockComponent.propTypes = {
+  styling: PropTypes.string.isRequired,
+};
+
+let workingConsoleError;
+
+/*
+ * Before/After each test we temporarily assign console.error to a mock function,
+ * so that we don't get ugly red warnings in the terminal when running tests!
+ * This allows us to check that we would be throwing errors correctly in real life,
+ * such as if someone passed an incorrect argument to the Col method!
+ */
+beforeEach(() => {
+  // NB. **ALWAYS** remember to reset the mocked console messages back to their original state!!
+  workingConsoleError = console.error;
+  console.error = jest.fn();
+});
+
+afterEach(() => {
+  console.error = workingConsoleError;
+});
+
+describe('Grid : Col(n) method', () => {
+  it('should render via Emotion as CSS when passed as a className to a React component', () => {
+    const test = Col(2);
+    const wrapper = shallow(<MockComponent styling={test} />);
```

Comment:
As above: rather than a `styling` prop, add a `css={test}` prop here. This will get processed by Emotion, and passed through to `MockComponent` as a `className`.

Reply:
I *think* i'm on the same page as you but it is probably worth having an in-person chat. There's plenty of things I can fix up and address in the meantime tho :)

Reply:
The tl;dr of what I was suggesting is, you could replace
```
    const wrapper = shallow(<MockComponent styling={test} />);
```
with
```
    const wrapper = shallow(<div css={test} />);
```
and make this test easier to understand.